core/domain/collection_services_test.py

```diff
@@ -824,20 +824,29 @@ def test_get_collection_summaries_with_deleted_collections_multi(
             ([], None))
 
     def test_search_collection_summaries(self) -> None:
+        # This test was originally based on exact matching, but with
```

Comment:
No, tests should be exact. If you expect more matches then have assertions for those as well and leave comments explaining why they show up. "Fuzzy" is still deterministic, it doesn't mean "random".